### ISSUE TITLE
A11y:  Healthcare widgets: change buttons to links in 4 non-scheduling widgets

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-cms#14219]
**STATUS: need to see Appointments team's designs and get them reviewed by Laura for a11y concerns listed below; requested on 6/29 via [slack](https://dsva.slack.com/archives/CMNQT72LX/p1688059598863849)**

### User story

**AS A** Veteran
**I WANT** links in the logged-in version of Healthcare widgets to look like links, not buttons
**SO THAT** they are more semantic and so that my screen reader will handle them appropriately.

Related to two a11y tickets in team repo – both are related to the Schedule-appointment widget which we don't need to handle because that team is doing it (however, they apply equally to the other 4 widgets): 
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/55859
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/55861

Pages where these widgets are found:
- [refill prescription](https://www.va.gov/health-care/refill-track-prescriptions/)
- [get VA medical records](https://www.va.gov/health-care/get-medical-records)
- [send secure message](https://www.va.gov/health-care/secure-messaging)
- [view lab or test results](https://www.va.gov/health-care/view-test-and-lab-results)
- ~~[make appointment](https://www.va.gov/health-care/schedule-view-va-appointments)~~ _we don't need to do this one, as VAOS/Appointments team is doing it_

### Design and outstanding design questions

Assets from the Appointment team's progress:
- Appointment team's [Figma](https://www.google.com/url?q=https://www.figma.com/file/iqjkZx2EbrZu6leL6YfN45/Cerner-Handoff?type%3Ddesign%26node-id%3D1631-22727%26mode%3Ddesign%26t%3DtRNmDsnBT4mNkLSQ-4&sa=D&source=calendar&ust=1691190654755009&usg=AOvVaw0Amhfj9opI3tIKigkaGlZu) for the schedule-appointment widget. 
  - It has been updated to show a "templatized" view of the widget, which is a good first pass at what props would need to be added to the react widget/component.
- Appointment widget [CAIA intake ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/59293), with discussion over content

Questions for new design:
- Does a11y require each link to be unique?
- How are we using primary and secondary CTA styles? (I suspect they all have to be secondary, old button design was wrong)
- Should we use all their content? (Ask CAIA)
- Is it true (per their design) that we can provide a single link for all Cerner systems? If so it seems ridiculous that our widgets have a separate button per Cerner-using system/location
  - _Validate: whether all the buttons have the same link using a Cerner test user in Staging_

<detail>
<summary>Example screen shot of current widget with buttons:</summary>

<img width="281" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/4054752/b74157a0-d14e-477a-b54d-536a97f50590">

[Mural of Cerner widget flows](https://app.mural.co/t/departmentofveteransaffairs9999/m/vagov6717/1651166973924/bc8bc44cab4e4ae7b84819d22ba89b11aa70e5e1?sender=uebde62ceb342c6477fb09406)

</detail>

## Acceptance criteria

- [ ] Links to external portals (Cerner and My HealtheVet/VistA) appear as CTA links
  - [ ] CTA styles are as designed
  - [ ] Links have unique attributes/text as required by a11y
- [ ] A11y review completed

**Comment by wesrowe** [comment]
I reached out to the Appointments team to get their design artifact and see if it's ready for us to leverage.

**Comment by jilladams** [comment]
I suspect we can close this ticket by EOQ4. Per Conlon, the widgets now have owners who will be making this change along with MHV on VA.gov path changes. I could be wrong, but it will shake out / be verifiable in Q4.

**Comment by jilladams** [comment]
Thanks Fran - good call. Also: hilarious given that I filed this issue yesterday which is basically the same outcome, but starting from a different defect point: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17560. So: it'll get done, by us probably, unless the CTA widget transitions to Health Apps folks meantime.
